src/raiden_libs/constants.py

```diff
@@ -3,4 +3,4 @@
 UDC_SECURITY_MARGIN_FACTOR_MS: float = 1.1
 UDC_SECURITY_MARGIN_FACTOR_PFS: float = 2
 
-MATRIX_START_TIMEOUT = 10
+MATRIX_START_TIMEOUT = 30
```

Comment:
can you add a comment, that this is seconds?

Reply:
and do we have a test for that?

Reply:
A test for what?